app/src/main/java/com/ivy/wallet/ui/onboarding/OnboardingScreen.kt

```diff
@@ -44,35 +49,47 @@ fun BoxWithConstraintsScope.OnboardingScreen(screen: Screen.Onboarding) {
         )
     }
 
-    UI(
-        onboardingState = state,
-        currency = currency,
-        opGoogleSignIn = opGoogleSign,
+    LazyColumn(
+        modifier = Modifier
+            .fillMaxSize()
+            .statusBarsPadding()
+            .navigationBarsPadding()
+    ) {
 
-        accountSuggestions = accountSuggestions,
-        accounts = accounts,
+        item {
+            UI(
```

Comment:
UI() is supposed to be the root @Composable so nesting it in LazyColumn isn't correct. My suggestion it to add LazyColum inside the UI().

```
LazyColumn {
   when {
      ....
   }
}
```

Reply:
alrighty

Reply:
this PR doesn't scale...the app crashes when the screen gets to the set currency
so i would close it